exampleFolder/index.js

```diff
@@ -0,0 +1,52 @@
+const express = require('express');
+var app = express();
+const axios = require('axios');
+const { response } = require('express');
+
+//POST Route (/addPerson)
+app.post('/addPerson', (req, res) => {
+    console.log(req.body)
+    const body = req.body
+
+    axios.post('http://java-sample-api-2020.herokuapp.com/addPerson',body)
+    .then((res) => {
```

Comment:
'res' should be a reserved name for the post method because it was initialized in the parameters at the beginning of the function. It would be fine to use 'response' instead of 'res'.

Reply:
Great catch Jonathan. I agree here. You may run into issues with javascript not being able to tell one 'res' variable from the other. Consider renaming.

Reply:
Made changes